src/clusterfuzz/_internal/bot/fuzzers/afl/launcher.py

```diff
@@ -517,6 +548,32 @@ def fuzzer_stderr(self):
       self._fuzzer_stderr = ''
     return self._fuzzer_stderr
 
+  def set_environment_variables(self):
+    """Sets environment variables needed by afl."""
+    # Tell afl_driver to duplicate stderr to STDERR_FILENAME.
+    # Environment variable names and values that must be set before running afl.
+    stderr_file_path = self.stderr_file_path
+    if environment.is_android():
+      stderr_file_path = android.util.get_device_path(self.stderr_file_path)
+
+    environment.set_value(constants.FORKSRV_INIT_TMOUT_ENV_VAR,
+                          constants.FORKSERVER_TIMEOUT)
+    environment.set_value(constants.FAST_CAL_ENV_VAR, 1)
+    environment.set_value(constants.IGNORE_UNKNOWN_ENVS_ENV_VAR, 1)
+    environment.set_value(constants.SKIP_CRASHES_ENV_VAR, 1)
+    environment.set_value(constants.BENCH_UNTIL_CRASH_ENV_VAR, 1)
+    environment.set_value(constants.SKIP_CPUFREQ_ENV_VAR, 1)
+    environment.set_value(constants.STDERR_FILENAME_ENV_VAR,
+                          stderr_file_path)
+
+  def get_afl_environment_variables(self):
```

Comment:
This probably works but ugh it is hacky.

Reply:
I know, I'm very open to any alternative way to set our AFL environment variables over ADB though.